server/elm/elm-watch.sh

```diff
@@ -20,7 +20,7 @@ function run {
   date -R;
   echo -en "${COLOR_OFF}";
 
-  elm make src/Main.elm --output ../hedley/modules/custom/hedley_ncda/js/elm-main.js
+  elm make src/Main.elm --output ../hedley/modules/custom/hedley_general/js/elm-main.js
```

Comment:
Don't we need both?

Reply:
No. Both NCDA and Reports running under common Elm application (different pages).